### PR TITLE
[CDAP-7783] - Feature/start confirmation modal

### PR DESCRIPTION
**Pull request by PGuimarais**
screen shot 2016-12-12 at 12 04 15 pm

Adds confirmation modal to start / stop program entity types 

Bamboo: 
http://builds.cask.co/browse/CDAP-DRC5051

### CODE REVIEW

File: cdap-ui/app/cdap/text/text-en.yaml [hunk 1]
```diff
@@ -99,6 +99,10 @@ features:
     sendEventsButtonLabel: Send
     sendEventsClickLabel: Click to input events
     sendEventsFailed: Error - Send Events Failed
+    startConfirmLabel: Start
+    stopConfirmLabel: Stop
+    startConfirmation: Start *_{entityId}_*?
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/text/text-en.yaml]: This definitely needs to be more descriptive. 

> Are you sure you want to start the program: **ProgramName**

would be more appropriate.

Comment (PGuimarais) [reply cdap-ui/app/cdap/text/text-en.yaml]: fixed

File: cdap-ui/app/cdap/components/FastAction/StartStopAction/index.js [hunk 2]
```diff
@@ -58,34 +89,70 @@ export default class StartStopAction extends Component {
...
+
     if (this.state.status === 'RUNNING' || this.state.status === 'STARTING') {
       icon = 'fa fa-stop text-danger';
+      confirmBtnText = "stopConfirmLabel";
+      headerText = "Stop Program";
+      confirmationText = T.translate('features.FastAction.stopConfirmation', {entityId: this.props.entity.id});
     } else {
       icon = 'fa fa-play text-success';
+      confirmBtnText = "startConfirmLabel";
+      headerText = "Start Program";
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/FastAction/StartStopAction/index.js]: This should come from i18n too.

**Comment by ajainarayanan** [comment]
screen shot 2016-12-12 at 3 20 59 pm

1. Error overflows
    - It should have the same behavior of Showing "Error while starting the program" as title text and there should be a down arrow that will open up the entire message from the backend. This should be similar to any other error message we show in a wizard.
2. I initially opened the confirmation modal to start the program. 
    - I was immediately shown the error message when I started it with spinner.
    - After sometime the spinner changed the text to say that "`Are you sure you want to stop the program: ...`"

It should just show the error message at the bottom and the start button disabled and cancel button enabled.

**Comment by ajainarayanan** [comment]
👍 LGTM

**Pull request merged**